ethereum/erc20/view_token_balances.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW erc20.view_token_balances AS
+SELECT 
+wallet_address,
+token_address,
+t.symbol as token_symbol,
+amount_raw,
+amount_raw / 10^coalesce(t.decimals, 0) amount_formatted,
```

Comment:
Should maybe be `amount_raw / 10^coalesce(t.decimals, NULL) amount_formatted,`?

Think it is a bit weird when `amount_formatted` is actually raw 
![image](https://user-images.githubusercontent.com/32701246/123284199-5c4ff580-d50c-11eb-8774-4f3f36fabc69.png)

Would also maybe name this column just `amount`

Reply:
Yea that makes sense I'll update

Reply:
Updated this, only thing is that `amount` could be `null` if decimals don't exist, `amount_raw` is always set.

Reply:
I don't think `coalesce(var, NULL)` makes any sense. Should just be `amount_raw / 10^t.decimals` imo; indeed resulting in `NULL` value for `amount` in case no decimals are known.